src/stories/button/Buttons.style.js

```diff
@@ -0,0 +1,52 @@
+
+import styled, { css } from "styled-components";
+import theme from "../../theme";
+
+
+export const StyledButtonComponent = styled.button`
+  text-transform: uppercase;
+  border: 2px solid ${theme.white};
+  font-style:italic;
+
+
+  font-size: 1rem;
+  font-weight: 600;
+  background-color: ${({ choosecolor }) =>
+    choosecolor === "secondary" ? theme.basiccolor : theme?.[choosecolor]};
+
+
+  color: ${({ choosecolor }) =>
+    choosecolor === "secondary" ? theme?.[choosecolor] : theme.basiccolor};
```

Comment:
The variable name "choosecolor" isn't clear. Could you consider renaming it to something more descriptive and relevant?